packages/opensearch-config-logstash-filters/packaging

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+RAKE_VERSION=$(find ruby -maxdepth 1 -name 'rake-*' | sed 's/ruby\/rake-\(.*\)\.gem/\1/' | head -1)
+
+# shellcheck disable=1090
+source "${BOSH_PACKAGES_DIR:-/var/vcap/packages}/ruby-3.1/bosh/compile.env"
+
+gem install "ruby/rake-${RAKE_VERSION}.gem" --local
+
+opensearch_config/bin/build
+
+cp opensearch_config/target/* "${BOSH_INSTALL_TARGET}"
```

Comment:
I am confused where these lines come from, because I don't see them in the corresponding Logsearch packaging script

https://github.com/cloud-gov/logsearch-for-cloudfoundry/blob/develop/packages/logsearch-config-logstash-filters/packaging

Reply:
Yeah, I must have gotten confused with too many files open. Closing